vllm/v1/worker/block_table.py

```diff
@@ -13,29 +15,62 @@
 
 class BlockTable:
 
-    def __init__(
-        self,
-        block_size: int,
-        max_num_reqs: int,
-        max_num_blocks_per_req: int,
-        max_num_batched_tokens: int,
-        pin_memory: bool,
-        device: torch.device,
-    ):
-        self.block_size = block_size
+    def __init__(self,
+                 block_size: int,
+                 max_num_reqs: int,
+                 max_num_blocks_per_req: int,
+                 max_num_batched_tokens: int,
+                 pin_memory: bool,
+                 device: torch.device,
+                 kernel_sizes: Union[list[int], None] = None):
```

Comment:
```suggestion
                 kernel_block_size: int):
```

I prefer to deside the kernel_block_size in gpu_model_runner and pass a single int here.